server/search/routes/searchController.test.ts

```diff
@@ -0,0 +1,60 @@
+import { Express } from 'express'
+import request from 'supertest'
+import FindAndReferService from '../../services/findAndReferService'
+import { appWithAllRoutes } from '../../routes/testutils/appSetup'
+import serviceUserDetailsFactory from '../../../testutils/factories/serviceUserDetails'
+
+jest.mock('../../services/findAndReferService')
+jest.mock('../../data/hmppsAuthClient')
+
+const hmppsAuthClientBuilder = jest.fn()
+const findAndReferService = new FindAndReferService(hmppsAuthClientBuilder) as jest.Mocked<FindAndReferService>
+
+let app: Express
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+beforeEach(() => {
+  app = appWithAllRoutes({
+    services: {
+      findAndReferService,
+    },
+  })
+})
+
```

Comment:
added a temporary test so the file doesn't fail for having no tests.
The 2 commented tests below are because the endpoint it currently commented out until the feature is ready, meaning these will fail if run.